spec/requests/post_controller_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :request do
+  describe 'GET #index' do
```

Comment:
> * [ ]  We should use `type: request` as well done in the `spec/request/post_controler_spec.rb`. You can check this [article](https://www.microverse.org/blog/understanding-rspec-controller-request-specs-integration-test-using-capybara#toc-controller-spec-http-request) for help.
> 
 
- [ ] As requested by the previous reviewer 🙃 Please you need to update the Post spec as well. ✍🏻

Reply:
@BenMukebo, Please check properly and see that request is there, So what are you talking about?